nin/frontend/app/scripts/services/ninrc.js

```diff
@@ -0,0 +1,9 @@
+function ninrc($http) {
+  const xhr = new XMLHttpRequest();
```

Comment:
`fetch` totally has more than good enough browser support for use in nin now, btw.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

EDIT: Looking at this again, I realize it is implemented synchronously. If it need to be sync, I guess it's best to just leave it like it is.